gcp/basic-gke-cluster/README.md

```diff
@@ -47,51 +47,51 @@ To run the YAML that this blueprint generates, you need:
 
 ## Information required
 
-This blueprint requires:
+This blueprint requires a:
 
-* GCP project ID (see section below for instructions)
-* A GCP region
+* GCP project ID
+* GCP region
 
-### Creating Google Cloud project
+### Create a Google Cloud project
 
 #### Authenticate to gcloud
-Before configuring gcloud CLI you can check available Zones and Regions nearest to your location
+Before configuring gcloud CLI you can check for available Zones and Regions nearest to your location:
```

Comment:
Before configuring **the** gcloud CLI.

I would format `gcloud`, since it's the name of a command-line tool. I'm also not sure about capitalizing Zones and Regions; seems like they should be lowercase.

Reply:
changed to "Before running `gcloud init`, you can check for...